src/App.css

```diff
@@ -28,6 +28,18 @@
   color: #61dafb;
 }
 
+.Gd-header {
+  background-color: #4daa5d;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.Gd-logo {
+  max-height: 10vh;
```

Comment:
A common practice for images is to have their height derived by their width with their natural aspect ratio. So for example, if an image is 300x150px, and you force it to be 100px in width then its height would always be 50px. 

because of that, it's not very common to add CSS that specifically sets the height, it's mostly always driven by the width - so it may be worth giving that a go!